src/Yaapii.Xml/XMLSlice.cs

```diff
@@ -38,7 +38,8 @@ namespace Yaapii.Xml
     /// A XML which will slice when you call Nodes(xpath).
     /// </summary>
     public sealed class XMLSlice : IXML
-    {
+    {
+        private readonly IScalar<IXmlNamespaceResolver> context;

```

Comment:
💅🏻 you could order the members in the variables definition and inside the ctor like the order in the ctor call.